### PR TITLE
Adding OrganizerOnly as a valid option

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/6298

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ Accept wildcard characters: False
...
+Determines what types of participants will automatically be added to meetings organized by this user. Set this to EveryoneInCompany  if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. Set this to Everyone if you'd like to admit anonymous users by default. Set this to EveryoneInSameAndFederatedCompany if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. Set this to OrganizerOnly if you would like that only meeting organizers can bypass the lobby.  This setting also applies to participants joining via a PSTN device (i.e. a traditional phone).
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:276]:
```suggestion
Determines what types of participants will automatically be added to meetings organized by this user. Set this to EveryoneInCompany if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. Set this to Everyone if you'd like to admit anonymous users by default. Set this to EveryoneInSameAndFederatedCompany if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. Set this to OrganizerOnly if you would like that only meeting organizers can bypass the lobby. This setting also applies to participants joining via a PSTN device (i.e. a traditional phone).
```

**Comment by tseward** [comment]
@get-itips I think this would be easier to read if it was in bullet point/list format, i.e.:

- EveryoneInCompany [....]
- Everyone [...]

etc.

Thoughts?

**Comment by dariomws** [comment]
Yes, it could be easier to identify possible values. What do you think @get-itips?

**Comment by get-itips** [comment]
@tseward Trevor, I applied your suggestions, can you check and approve ? Thanks

**Comment by get-itips** [comment]
@Kateyanne Please merge, thanks

**Pull request merged**